sunpy/net/dataretriever/tests/test_eve.py

```diff
@@ -1,36 +1,138 @@
 import pytest
+from unittest import mock
+from datetime import datetime, timedelta
 
 from sunpy.time import parse_time
 from sunpy.time.timerange import TimeRange
-from sunpy.net.vso import VSOClient
+from sunpy.net.vso.vso import VSOClient
 from sunpy.net.vso.attrs import Time, Instrument, Source, Level
 from sunpy.net.dataretriever.client import QueryResponse
 import sunpy.net.dataretriever.sources.eve as eve
 from sunpy.net.fido_factory import UnifiedResponse
 from sunpy.net import Fido
+from sunpy.net import attr
 from sunpy.net import attrs as a
 
 
 LCClient = eve.EVEClient()
+BASEURL = eve.BASEURL
+
+
+def create_url(start, end):
+    """
+    This function creates a url based on the EVEClient data,
+    instead of making an online request.
+    """
+
+    start = datetime.strptime(start, '%Y/%m/%d')
+    end = datetime.strptime(end, '%Y/%m/%d')
+
+    lst = list()
+    for n in range(int((end - start).days)+1):
+        lst.append((start+timedelta(n)).strftime(BASEURL))
+    return lst
+
+
+def mock_query_object(start_date, end_date, level=0):
+    """
+    Creation of a QueryResponse object, and prefill some
+    downloaded data from eve.EVEClient().fetch(Time('20 ..)
+    """
+    # Create a mock QueryResponse object
+    map_ = {
+        'TimeRange': TimeRange(parse_time(start_date), parse_time(end_date)),
```

Comment:
I think you can drop the `parse_time` call here. `TimeRange` is calling `parse_time` if I remember correctly.